docusaurus/docs/reactnative/ui-components/message_footer.mdx

```diff
@@ -6,6 +6,7 @@ title: MessageFooter
 import Members from '../common-content/contexts/channel-context/members.mdx';
 
 import Alignment from '../common-content/contexts/message-context/alignment.mdx';
+import DeletedMessagesVisibilityType from '../common-content/contexts/messages-context/deleted_messages_visibility_type.mdx'
```

Comment:
you can replace this with import from channel/props directly. Imports should be from their original source.

Reply:
but keep it in alfabetical order

Reply:
Oh, I thought that we should have the same pattern as we have for others, that is why I did this way. Anyways okay, will do it now.